crates/evm/traces/src/decoder/mod.rs

```diff
@@ -533,23 +559,23 @@ impl CallTraceDecoder {
     }
 
     /// Decodes an event.
-    pub async fn decode_event<'a>(&self, log: &'a LogData) -> DecodedCallLog<'a> {
-        let &[t0, ..] = log.topics() else { return DecodedCallLog::Raw(log) };
+    pub async fn decode_event<'a>(&self, log: &'a CallLog) -> DecodedCallLog<'a> {
```

Comment:
why was this changed?

Reply:
good point, changed it back so it again only passes down what is required i.e. `LogData`, not the entire `CallLog` that includes the decoded fields